common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/packets/EntityPackets.java

```diff
@@ -146,6 +147,25 @@ public void registerMap() {
             }
         });
 
+        protocol.registerClientbound(ClientboundPackets1_12_1.ENTITY_EFFECT, new PacketRemapper() {
+            @Override
+            public void registerMap() {
+                map(Type.VAR_INT); // Entity id
+                map(Type.BYTE); // Effect id
+                map(Type.BYTE); // Amplifier
+                map(Type.VAR_INT); // Duration
+
+                handler(packetWrapper -> {
+                    byte flags = packetWrapper.read(Type.BYTE); // Input Flags
+
+                    if (Via.getConfig().isNewEffectIndicator())
+                        flags += 2;
```

Comment:
Looking at wiki.vg it looks like a bitmask so, I think this should be `flags |= 0x02` to make sure this does not shift anything else.

Reply:
|= 0x02 does not work, besides there is no other use case anyway, have you tried if your approach works?

Reply:
fixed

Reply:
After looking at source code it looks like 0x04 is the right mask to apply 👍